src/API/Exception/ZoneSettingFailException.php

```diff
@@ -0,0 +1,6 @@
+<?php
+namespace CF\API\Exception;
+class ZoneSettingFailException extends CloudFlareException
+{
+    protected $message = 'Oops, something went wrong, please try again in a few minutes';
```

Comment:
These messages are not localized. We should find a solution to make them localized.

Also not relevant with this PR but we should support links in notification since 
`cloudflare.com/a/page-rules` 
will become 
`cloudflare.com/a/page-rules?utm_source=aNewPluginWithLongTextHereWhichWillLookReallyUgly`